apps/typeform/frontend/src/constants.ts

```diff
@@ -2,6 +2,6 @@ const SDK_WINDOW_HEIGHT = 450;
 const BASE_URL = 'https://api.typeform.com';
 
 // TODO Move to env var
-const CLIENT_ID = 'HC3UDnoiaP1UCMqJ7kCAyTFdHrDt8nLtXx4BKRJxom2M';
+const CLIENT_ID = '2vyzzT2AjqrtfWKmaigvZjF8oYwUXrJABmcS5WK4MPJg';
```

Comment:
is this intentional?

Reply:
Thanks actually I should move that back. There's some weird stuff in the instructions where this client ID is pointed to the prodcution app. but why is it just hardcoded like this in the first place? haha there's a TODO around moving stuff to env variables which seems like we should do that eventually but didn't yet. I'd also like to test this one better on staging before merging so let's chat about that later too.

Reply:
changed that back and ready for rereview @mgoudy91